README.rst

```diff
@@ -1427,76 +1386,75 @@ variables.
                     -p 127.0.0.1:8082:8082 \
                     myapp:1.0.0
 
-To check the instance logs, say:
+To check the instance logs, run:
 
 .. code-block:: bash
 
     docker logs instance-1
 
-******************
 Runtime image tag
-******************
+^^^^^^^^^^^^^^^^^
 
-The result image is tagged as follows:
+The final image is tagged as follows:
 
-* ``<name>:<detected_version>[-<suffix>]``: by default;
-* ``<name>:<version>[-<suffix>]``: if the ``--version`` parameter is specified;
-* ``<tag>``: if the ``--tag`` parameter is specified.
+* ``<name>:<detected_version>[-<suffix>]`` by default.
+* ``<name>:<version>[-<suffix>]`` if the ``--version`` parameter is specified.
+* ``<tag>`` if the ``--tag`` parameter is specified.
 
 .. _cartridge-cli-build-and-runtime-images:
 
-*************************
 Build and runtime images
```

Comment:
The reader will only know what build and runtime images are when reaching line 1448 of the text. The terms "build image" and "runtime image" have already been mentioned multiple times. This is confusing, so the text needs restructuring. What do you think, @NickVolynkin?